src/script/interpreter.cpp

```diff
@@ -1166,30 +1166,8 @@ bool TransactionSignatureChecker::CheckSig(const vector<unsigned char>& vchSigIn
     return true;
 }
 
-bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime, bool fSequence) const
+static bool SomeCheckLockTime(const CScriptNum& nLockTime, const int64_t txToLockTime)
```

Comment:
Probably want a better name,  but that's fine.  Probably doesn't need to be static either.